partcad/src/partcad/telemetry.py

```
@@ -0,0 +1,125 @@
+
+                for key, value in kwargs.items():
+                    if isinstance(value, (str, int, float, bool)):
+                        span.set_attribute(key, value)
+                return await func(*args, **kwargs)
+
+        return wrapper
+
+    return decorator
+
+
+def instrument(exclude: list = [], attr_getters=lambda attr_value: lambda *args, **kwargs: {}):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)